### ISSUE TITLE
Lao LAK Currency is not display correct as decimal value on the Pay Button in presentPaymentSheet

### ISSUE THREAD

**Issue by TG7Dev** [flutter-stripe/flutter_stripe#1380]
Hi Team,
I'm a developer from Laos. when I tested LAK payment from flutter app for example 50000 LAK. I understand that I need to mutiply it by 100 in order to success the transaction. so the amount willbe 5000000LAK passed to stripe back, the backend worked properly and it responded back to presentPaymentSheet() where frontend user will fill up the visa card no. card holder name and with the amount display to user.Unfortunately, it seems like the flutter_stripe library not recognize LAK currency.as LAK amount showed as LAK 5,000,000 insteadof LAK 50,000.00 as it should be that would lead a confusion to a frontend user. However when I tested USD, it showned as  $50,000.00 as it expected.

To sum up, LAK display amount must same as USD logic so that it will make sense to frontend user.
![Screenshot_20230822-121235](https://github.com/flutter-stripe/flutter_stripe/assets/134901186/f1f1c372-8334-49b5-ad7f-1de3d24a0f6d)
![Screenshot_20230822-121323](https://github.com/flutter-stripe/flutter_stripe/assets/134901186/3f236dc2-6b0f-4d1a-b250-0dda03b545dd)

**Comment by remonh87** [comment]
Can you check the value in the Stripe payment intent in the dashboard? I think this is not a bug in the frontend but something is wrong in the payment intent. The amount displayed on the payment sheet is outside our control.
